app/templates/components/tabbed-navigation.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if device.isMobile}}
-  <div role="button" class="ui segment center aligned" {{action 'toggleMenu'}}>
+  <div role="button" class="ui segment center aligned" onClick={{action 'toggleMenu'}}>
```

Comment:
Why is the value not getting updated ? `onClick={{action 'toggleMenu'}}` and `{{action 'toggleMenu'}}` are basically the same internally. Please investigate what is causing any issue here.